src/test/java/storage/MemoryProductStorageTest.java

```diff
@@ -0,0 +1,77 @@
+package storage;
+
+import java.util.UUID;
+
+import store.MemoryProductStorage;
+import store.NotExistingProductException;
+import store.Produit;
+
+
+public class MemoryProductStorageTest{
+
+	MemoryProductStorage store = new MemoryProductStorage();
+	
+	// empty storage returns empty list
+	
+	public boolean emptyList() {
+		
+		return store.all().isEmpty();
+	}
+	
+	// adding a product returns the product in the list
```

Comment:
Why adding a comment to restate the name of the method?

Reply:
Yes use "adding a product returns the product in the list" as the name of your function !
even better, you get the best of both !